pages/[id].jsx

```diff
@@ -0,0 +1,12 @@
+import { useRouter } from 'next/router';
```

Comment:
I asked Meg to include a <Head> component in her PR, so I think we should be consistent and have one here as well, so we'll want to `import Head from 'next/head';` here.

Reply:
Good suggestion. I added it in